src/app/core/components/payment-overview/payment-overview.component.ts

```diff
@@ -150,7 +150,7 @@ export class PaymentOverviewComponent implements OnInit {
         if (data.hasOwnProperty('bar_user_full_name')) {
           model.userFullName = data.bar_user_full_name;
         }
-        model.userRole = UserRole.SRFEECLERK;
+        model.userRole = UserRole.SRFEECLERKDISPLAY;
```

Comment:
refactor all this code. there shouldn't be any if else here. it should get current User and then get its role as a property like getCurrentUser.getRole().getRoleDescription